engines/bastion_katello/app/assets/javascripts/bastion_katello/hosts/host-bulk-action.factory.js

```diff
@@ -26,7 +26,8 @@ angular.module('Bastion.hosts').factory('HostBulkAction',
             releaseVersion: {method: 'PUT', params: {action: 'release_version'}},
             availableIncrementalUpdates: {method: 'POST', isArray: true, params: {action: 'available_incremental_updates'}},
             moduleStreams: {method: 'POST', params: {action: 'module_streams'}},
-            traces: {method: 'POST', params: {action: 'traces'}}
+            traces: {method: 'POST', params: {action: 'fetch_traces'}},
+            resolve: {method: 'PUT', isArray: true, params: {action: 'resolve_traces'}}
```

Comment:
I was expecting to see this called `resolve_traces` but it's still called `resolve` (you changed only the server side, and the previous name of `traces` was fine - the 'fetch' is essentially implied)

Reply:
Updated, I was trying to be fancy with the fetch but I see now what you mean. Thanks for ruining my fun :laughing: